.github/ISSUE_TEMPLATE.md

```diff
@@ -1 +1,5 @@
-<!-- Before opening a new issue, please check that no existing issue covers the same topic -->
+<!-- Before opening a new issue, please check that no existing issue covers the same topic
+
+
+Our contributing guidelines (link given above) contain some templates to be used when posting a new issue. We encourage you to use them as appropriate.
```

Comment:
I'm being extremely picky here 😛 but can you:
- Add a line break (not empty line) between the two sentences "Our contributing..." and "We encourage..."?
- Change the double empty line separating "Before opening..." and "Our contributing..." to just single empty line?